### PR TITLE
Added toggler to set email as replyTo

### PR DESCRIPTION
**Pull request by Quehnie**

### CODE REVIEW

File: Form/Type/DynamicFormType.php [hunk 1]
```diff
@@ -339,6 +339,20 @@ public function getNotifyToMailAddress($formData = [])
...
      */
+    public function getNotifyReplyToMailAddress($formData = [])
+    {
+        if ($this->getTranslation()->getReplyTo()) {
+            if ($formData->__isset('email')) {
+                return $formData->__get('email');
+            }
+        }
+
+        return $this->getAttribute('mail_notify_replyto_address');
```
Comment (alexander-schranz) [inline Form/Type/DynamicFormType.php]: call here: `return parent::getNotifyReplyToMailAddress($formData);`

File: Resources/translations/sulu/backend.fr.xlf [hunk 2]
```diff
@@ -34,6 +34,10 @@
                 <source>l91_sulu_form.deactivate_customer</source>
                 <target>déactiver les mails de confirmation</target>
             </trans-unit>
+            <trans-unit id="9">
+                <source>l91_sulu_form.user_as_reply_to</source>
+                <target>"répondre" compléter l'adresse automatiquement</target>
```
Comment (alexander-schranz) [inline Resources/translations/sulu/backend.fr.xlf:39]: what?

File: Resources/views/forms/template.html.twig [hunk 3]
```diff
@@ -228,6 +229,27 @@
                     </div>
                 </div>
             </div>
+
+            <div class="grid-row m-bottom-40">
+                <div class="grid-col-6">
```
Comment (alexander-schranz) [inline Resources/views/forms/template.html.twig]: Would do it like

Col 3                      | Col3              | Col3                    | Col 3                    
---------------- | ----------- | -------------- | -----------------
ReplyTo                  | Attchment      | Notify Email        | Customer Email    

Or if the space is not enough

Col 6 | Col6
------|------
ReplyTo | Attachment 
Notify Email | Customer Email

**Review by alexander-schranz — CHANGES_REQUESTED** [review]

File: Resources/translations/sulu/backend.fr.xlf (hunk 2 above)
Comment (Quehnie) [reply Resources/translations/sulu/backend.fr.xlf:39]: Asked aline for translation :) so it's correct baguette ^^

Comment (alexander-schranz) [reply Resources/translations/sulu/backend.fr.xlf:39]: ok missed ReplyTo but if aline say its correct "baguette"

**Review by alexander-schranz — APPROVED** [review]

**Pull request merged**